apps/calendar/test/unit/setup.js

```diff
@@ -26,350 +25,11 @@ function configureLoader() {
     }
   });
 
-  configured = true;
-}
-
-function l10nLink(href) {
-  var link = document.createElement('link');
-  link.setAttribute('href', href);
-  link.setAttribute('rel', 'localization');
-  document.head.appendChild(link);
-}
-
-function l10nMeta(defaultLanguage, availableLanguages) {
-  var metaDL = document.createElement('meta');
-  metaDL.setAttribute('name', 'defaultLanguage');
-  metaDL.setAttribute('content', defaultLanguage);
-
-  var metaAL = document.createElement('meta');
-  metaAL.setAttribute('name', 'availableLanguages');
-  metaAL.setAttribute('content', availableLanguages.join(', '));
-
-  document.head.appendChild(metaDL);
-  document.head.appendChild(metaAL);
-}
-
-function loadL10n() {
-  return new Promise((resolve) => {
-    requirejs(['shared/l10n'], () => {
-      requirejs(['shared/l10n_date'], () => {
-        // Massive hack to trick l10n to load
-        // TODO: upstream a fix to l10n.js
-        document.dispatchEvent(new Event('DOMContentLoaded'));
-
-        var readyState = navigator.mozL10n.readyState;
-        switch (readyState) {
-          case 'complete':
-          case 'interactive':
-            return resolve();
-        }
-
-        window.addEventListener('localized', () => resolve());
-      });
-    });
-  });
-}
-
-function loadApp() {
-  // TODO(gareth): We load these extras by default here since we need them
-  //     for the test helpers below, but this situation is really sad...
-  return new Promise((accept) => {
-    requirejs([
-      'app',
-      'core',
-      'db',
-      'ext/chai',
-      'ext/chai-as-promised',
-      'provider/factory',
-      'router',
-      'test/support/fake_page',
-      'test/support/factories/all',
-      'test/support/mock_provider'
-    ], () => accept());
-  });
-}
+  return new Promise(resolve => requirejs(['test/support/test_loader'], resolve));
```

Comment:
Now `setup.js` and sets `window.testAgentRuntime.testLoader` to load `test_loader` and then call `testLoader` with the param `path`